src/Command/Notify.js

```diff
@@ -0,0 +1,51 @@
+const Command = require('../Command');
+
+const messages = {
+    noGame: 'You must specify one of the available notifications to subscribe to: {types}',
+    notifyGameSeparator: 'NOTIFY_GAME_SEPARATOR',
+    cantGrant: 'I was unable to grant you the role.',
+    success: 'Your role has been granted.'
+};
+
+module.exports = Command.extend({
+    config: null,
+    shouldDeleteMessage: true,
+    commandName: 'notifyme',
+    commandAliases: ['notifywarband', 'wbnotify', 'war', 'warbandnotify'],
+    dependencies: {
+        'config': 'config'
+    },
+    initialize: function (dependencyGraph) {
+        this._super(dependencyGraph);
+    },
+    processMessage: async function (message, tokens, commandName) {
+        let notifyGame;
+        if (this.commandAliases.indexOf(commandName) >= 0) {
+            notifyGame = 'warband';
+        } else if (tokens.length < 2) {
+            return message.channel.send(this.i18n.__mf(
+                messages.noGame,
+                {
+                    types: Object.keys(this.config.notificationRoles).join(this.i18n.__mf(messages.notifyGameSeparator))
+                }
+            ));
+        } else {
+            notifyGame = tokens[1].toLowerCase();
+        }
+
+        const roleId = message.guild.roles.cache.find(role => role.id === this.config.notificationRoles[notifyGame]);
+
+        let result;
+        message.member.roles.add(roleId)
+            .catch(error => {
+                console.error(error);
```

Comment:
Can `console.error` cause the bot to crash / become unresponsive until it is rebooted? I think we use `console.log` to log errors at the moment.

Reply:
Why would it?

Reply:
I recall that in nodemon, the process stops until a restart / change is applied, but I don't know if this extends to just `node`, hence us using `console.log` everywhere.